src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -223,7 +226,7 @@
     For Application permissions, please use "ad app permission admin-consent"
 examples:
   - name: Grant a native application with permissions to access an existing API with TTL of 2 years
-    text: az ad app permission grant --id e042ec79-34cd-498f-9d9f-1234234 --api a0322f79-57df-498f-9d9f-12678
+    text: az ad app permission grant --id e042ec79-34cd-498f-9d9f-1234234 --api a0322f79-57df-498f-9d9f-12678 --scope Directory.Read.All
```

Comment:
`--scope`'s default value `user_impersonation` is removed, as `user_impersonation` is very highly privileged.